src/List.elm

```diff
@@ -378,7 +378,7 @@ append xs ys =
 -}
 concat : List (List a) -> List a
 concat lists =
-  foldr append [] lists
+  foldr (++) [] lists
```

Comment:
I would rather suggest making `append` an alias for `(++)` so the performance is consistent

Reply:
`++` is currently defined in terms of `append`: https://github.com/elm/core/blob/master/src/Basics.elm#L81 so I can't do that without further changes to the codebase.

Reply:
I think that ` Elm.Kernel.Utils.append` and `List.append` are two independent things but I haven't check it to be 100% sure.

Reply:
Oh, you're right. Let's wait for opinion from some of the maintainers before doing larger changes.

Reply:
this will have a similar fate as https://github.com/elm/core/pull/631

Reply:
I modified the PR to use `Basics.++` in `List.append` instead of in `List.concat`

Reply:
I’ll take a look at this during the weekend.